hpobench/benchmarks/surrogates/paramnet_benchmark.py

```diff
@@ -269,17 +291,22 @@ def objective_function(self, configuration: Union[CS.Configuration, Dict],
 
         return {'function_value': float(y),
                 'cost': fidelity['budget'],
-                'info': {'learning_curve': lc.tolist(),
-                         'observed_epochs': len(lc)}}
+                'info': {'fidelity': fidelity['budget'],
```

Comment:
L276 and L294 should be the same? I assume both should be `'info': {'fidelity': fidelity,`

Reply:
Thanks! Should I also add the entries: 

```
'learning_curve': []
'observed_epochs': 0
```
to the info dict of L274?

Reply:
Good point. Yes, please.